### DISCUSSION TITLE
Hidden section visible for owners into the Spaces without entering to edit

### DISCUSSION THREAD

**Discussion by utamayo** [GitbookIO/community#811] [Feature Requests]
When I am looking my spaces the hidden sections are not visible until I click to edit button, it is awkward procedure. I recomend to be visible for the owners with a lighter color fonts for example when we are in https://app.gitbook.com/xxxxxxx.
In my case for example I forgot what it is hidden so I have to edit see the hidden sections and then archive the edition.

**Comment by addisonschultz** [comment]
Yeah this is a great idea - I think especially in the editor this could be more clear!
